docs/introduction/README.md

```diff
@@ -0,0 +1,15 @@
+# Introduction
+
```

Comment:
what's the point of this file? it isn't added to the sidebar ... (config.js didn't change in this PR)

Reply:
I like all the sections having landing pages. Is that wrong? :P

Reply:
ah, and it's only accessible from the link, there's nothing to navigate to it within the sidebar.

Reply:
Right, hence my `TODO` in the opening comment for this PR:

> TODO: can we get the sidebar headers to be links to these?

Think that's possible?

Reply:
missed it, yup. spec has it, i'll update that.